snapshots/windows/cimfs.go

```diff
@@ -110,6 +106,10 @@ func (s *cimFSSnapshotter) isCimLayer(ctx context.Context, key string) (bool, er
 }
 
 func (s *cimFSSnapshotter) Usage(ctx context.Context, key string) (snapshots.Usage, error) {
+	if !cimfs.IsCimFSSupported() {
+		return snapshots.Usage{}, fmt.Errorf("host windows version doesn't support CimFS")
```

Comment:
Not sure why this is showing an extra space on github. On local editor, the spacing seems fine:

![image](https://github.com/containerd/containerd/assets/99994218/a600212d-012d-4348-96ea-7d5d7b7b2da9)